jep/7/README.adoc

```diff
@@ -67,22 +69,68 @@ endif::[]
 
 == Abstract
 
-Due to its negative impact on core maintenance, distribution of the unmaintained `ruby-runtime` plugin will be suspended.
-Distribution of plugins depending on it will be suspended until their dependency is removed.
+Due to its negative impact on core maintenance, distribution of the unmaintained
+link:https://github.com/jenkinsci/ruby-runtime-plugin[Ruby Runtime plugin] and
+link:https://github.com/jenkinsci/jenkins.py[Jenkins Python SDK] will be suspended.
+Distribution of plugins depending on them will be suspended until their dependency is removed.
 
 
 == Specification
 
-`ruby-runtime` will be added to the `artifact-ignores` file of the update center generator so it is no longer available for download on update sites.
-// Perhaps: Add security warning?
+Ruby runtime and Python SDK will be added to the `artifact-ignores` file of the update center generator, so it is no longer available for download on update sites.
+A warning will be added to the update center so that all users of the plugins receive a warning in advance.
+The plugins will be also marked for adoption so that users can step up and take over maintenance without a 2 weeks waiting delay.
 
-Plugins with a mandatory dependency on `ruby-runtime` will be added to the same file, as they will not be installable without `ruby-runtime` being available.
+Plugins with a mandatory dependency on the runtimes will be added to the same file, as they will not be installable without the respective runtimes being available.
+The multiple Git repositories holding the `ruby-runtime` and Python SDK source code will be archived on GitHub.
 
-The multiple Git repositories holding the `ruby-runtime` source code will be archived on GitHub.
+Support for Ruby Runtime in the Jenkins core will be removed in future Jenkins versions.
```

Comment:
does this not conflict with https://github.com/jenkinsci/jep/pull/356/files#diff-79416440f404d2b50d67e437cd226b4e742c20f94cce5a10f23be10d063d65adR80 ?

Reply:
I do not see a  conflict. Update site and the Jenkins core are separate components

Reply:
I see it conflicting as does `ruby-runtime` plugin which would be up for adoption based on line 82 above actually requires this support in core to work?  (maybe I am missing something, or my suggestion above clarifies this)

Reply:
Agree with James, let's not pretend it can come back once we've ripped out support.